velox/exec/tests/AggregationFuzzerTest.cpp

```diff
@@ -835,15 +841,23 @@ int main(int argc, char** argv) {
 
   size_t initialSeed = FLAGS_seed == 0 ? std::time(nullptr) : FLAGS_seed;
 
-  auto duckQueryRunner =
-      std::make_unique<facebook::velox::exec::test::DuckQueryRunner>();
-  duckQueryRunner->disableAggregateFunctions({
-      "skewness",
-      // DuckDB results on constant inputs are incorrect. Should be NaN,
-      // but DuckDB returns some random value.
-      "kurtosis",
-      "entropy",
-  });
+  std::unique_ptr<facebook::velox::exec::test::ReferenceQueryRunner> runner;
```

Comment:
nit: perhaps, extract the logic of createing a query runner into a helper function

```
auto queryRunner = setupReferenceQueryRunner();
```